### PR TITLE
Support *.cc file as a header

### PR DESCRIPTION
**Pull request by sharkpc0813**
Make PR for [issue](https://github.com/CocoaPods/CocoaPods/issues/6833)

`*.cc` file would be included exceptionally.

https://google.github.io/styleguide/cppguide.html#Self_contained_Headers

### CODE REVIEW

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! <br />
You can find it at <a href="https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

Here's an example of your CHANGELOG entry:

```markdown
* Support *.cc file as a header  
  [sharkpc0813](https://github.com/sharkpc0813)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

*note*: There are two invisible spaces after the entry's text.


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,9 @@
 
 ##### Enhancements
 
-* None.  
+* Support *.cc extension file as a header.
```
Comment (segiddins) [inline CHANGELOG.md]: Please place the `*.cc` in backticks

Comment (sharkpc0813) [inline CHANGELOG.md]: No problems 👍

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please rebase to get rid of the merge commits in this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +24,7 @@
 
 * None.  
 
+
```
Comment (dnkoutso) [inline CHANGELOG.md]: remove line

**Comment by dnkoutso** [comment]
Please run `rubocop -a`

**Comment by sharkpc0813** [comment]
Any suggestions? Do I request newly?

**Comment by dnkoutso** [comment]
/cc @orta any suggestions on the build failure?

```
1164 specifications (1910 requirements), 0 failures, 0 errors
Running RuboCop...
Inspecting 106 files
..........................................................................................................
106 files inspected, no offenses detected
The command "bundle exec rake spec:all" exited with 0.
1.82s$ if [ "$COCOAPODS_CI_TASKS" = "LINT" ]; then bundle exec danger; fi
Danger has failed
```

**Comment by orta** [comment]
The build error is in the comment above yours:

screen shot 2017-10-10 at 12 50 59 pm

**Pull request closed without merging**